### PR TITLE
fix: nullable translation attributes

### PR DESCRIPTION
**Pull request by Gaitholabi**
This PR revises the currently supported translation models and validates their correctness.

### CODE REVIEW

File: src/Model/Core/BrandTranslation.php [hunk 1]
```diff
@@ -62,23 +56,4 @@ public function setBrandId(int $brandId): self
...
-    /**
-     * @param string $locale
-     * @return self
-     */
-    public function setLocale(string $locale): self
-    {
-        $this->locale = $locale;
-
-        return $this;
-    }
```
Comment (bytestream) [inline src/Model/Core/BrandTranslation.php:83]: We should split changes in this file into a separate refactor PR.

File: src/Model/Shared/OptionTranslation.php [hunk 2]
```diff
@@ -20,18 +21,20 @@ class OptionTranslation extends BaseTranslation
...
     }
 
     /**
-     * @param int $ticketCustomFieldOptionId
+     * @deprecated
+     * @param int|null $ticketCustomFieldOptionId
      * @return self
      */
-    public function setTicketCustomFieldOptionId(int $ticketCustomFieldOptionId): self
+    public function setTicketCustomFieldOptionId(?int $ticketCustomFieldOptionId): self
```
Comment (bytestream) [inline src/Model/Shared/OptionTranslation.php:37]: Is there a bug reference / stack trace for this? I thought the issue was specifically to do with `null` `description` attributes on custom fields.

**Review by bytestream — CHANGES_REQUESTED** [review]
Thanks. It seems the `UserCustomFieldTranslation` model was missed in https://github.com/supportpal/api-client-php/pull/85.

File: src/Model/Shared/OptionTranslation.php (hunk 2 above)
Comment (Gaitholabi) [reply src/Model/Shared/OptionTranslation.php:37]: There isn't a reference, the PR revises all translation models and fixes them. The initiator was the null/description.
The issue I detected here was having OptionTranslation linked to the TicketCustomfield where infact it is not. I only kept the methods here, and added nullability so code doesn't crash, and not to break BC.

These methods should be removed in the next version

**Review by bytestream — APPROVED** [review]

**Pull request merged**